src/main/java/command/UnmarkCommand.java

```diff
@@ -0,0 +1,27 @@
+package command;
+
+import main.TaskList;
+import main.Storage;
+import main.DukeException;
+import main.Ui;
+import task.Task;
+
+public class UnmarkCommand extends Command{
+
+    int index;
```

Comment:
Similar to the comment in the MarkCommand class, do consider making this variable private.